backend/apps/owasp/api/search/chapter.py

```diff
@@ -39,11 +40,11 @@ def chapters(request):
     """Search chapters API endpoint."""
     page = int(request.GET.get("page", 1))
     query = request.GET.get("q", "")
-
+    active_chapters_count = IndexSynonymsMixin.get_algolia_index_count("local_chapters")
```

Comment:
The `local_*` naming will only work for local environment. 
Please update the base class to add environment prefix to the index name and pass only `chapters` (same for other indexed models).